rpc/nodeagent/compute/moc_nodeagent_virtualmachine.pb.go

```diff
@@ -1287,115 +1295,117 @@ func init() {
-	0x2e, 0x4c, 0x32, 0x18, 0xf0, 0x21, 0xf5, 0x5f, 0x0e, 0xf5, 0x95, 0x91, 0xfb, 0x54, 0x3a, 0x8d,
-	0xad, 0x99, 0xed, 0x06, 0x9e, 0x56, 0xa0, 0x1d, 0x80, 0x88, 0x0f, 0xa9, 0xd0, 0xcb, 0x53, 0x33,
-	0x4e, 0x56, 0xdb, 0x0f, 0x92, 0x71, 0x32, 0x16, 0xe3, 0x8c, 0x89, 0xce, 0x43, 0x5a, 0x21, 0xce,
-	0xd2, 0x9d, 0x79, 0x28, 0xad, 0x77, 0x3c, 0x46, 0xeb, 0x3c, 0xe8, 0x7b, 0x8b, 0x1a, 0x39, 0xcb,
-	0x99, 0x3c, 0x1c, 0x1a, 0x11, 0xb6, 0x2a, 0xd4, 0x81, 0x8d, 0x7e, 0x21, 0xc2, 0x84, 0xea, 0x8a,
-	0xa1, 0xda, 0x34, 0x98, 0xe3, 0x32, 0x0b, 0x5c, 0x0e, 0x44, 0x8f, 0x61, 0x56, 0x91, 0x9e, 0x74,
-	0x56, 0xcd, 0xa2, 0x0d, 0xe3, 0xe0, 0x15, 0xe9, 0x49, 0x6c, 0xc4, 0xe8, 0x2b, 0x00, 0xf3, 0x5c,
-	0xd8, 0xd5, 0xb1, 0x38, 0x0f, 0x8c, 0xd1, 0x23, 0x63, 0x74, 0x34, 0x16, 0xe7, 0x23, 0xca, 0xd8,
-	0xa3, 0xdf, 0xc3, 0xe6, 0xe4, 0xeb, 0x24, 0x94, 0x8a, 0x84, 0x1e, 0xbd, 0x60, 0x74, 0xe8, 0xbc,
-	0x67, 0x3c, 0x7d, 0x5c, 0x32, 0x71, 0xa6, 0x6c, 0x71, 0x85, 0x0f, 0x3d, 0x45, 0x9e, 0xe4, 0xb1,
-	0x93, 0x77, 0xa6, 0x7d, 0xf2, 0xbe, 0x84, 0x07, 0x79, 0x8b, 0xb7, 0x7c, 0x3e, 0x16, 0xd1, 0x68,
-	0xaf, 0xfc, 0x99, 0xfb, 0xa8, 0x24, 0x90, 0xca, 0x47, 0xef, 0x3f, 0x6a, 0xf0, 0x51, 0x25, 0x6f,
-	0xfb, 0xfa, 0x7d, 0xe7, 0xc4, 0xdf, 0xdd, 0x8b, 0xf7, 0xbf, 0xf5, 0x22, 0x7d, 0x1c, 0x87, 0xfb,
-	0x3c, 0x08, 0x48, 0xe8, 0xa7, 0x79, 0x3f, 0x2d, 0xce, 0x49, 0xfb, 0x84, 0xbc, 0x27, 0xfb, 0xe2,
-	0x90, 0xdd, 0x85, 0xf9, 0xcc, 0xc5, 0x3d, 0xbd, 0x9d, 0x54, 0x91, 0xe8, 0x7a, 0x82, 0x45, 0xca,
-	0xde, 0xe4, 0x2d, 0x10, 0x31, 0xf8, 0x60, 0x62, 0x71, 0x12, 0x46, 0xb1, 0xea, 0x10, 0x41, 0x02,
-	0xaa, 0x26, 0x17, 0xc6, 0x4f, 0xef, 0xf4, 0x9a, 0xc7, 0xe0, 0x6a, 0x6f, 0xc8, 0x85, 0x06, 0x8e,
-	0xc3, 0x5d, 0xa9, 0x2f, 0x9a, 0xb9, 0x6b, 0xe4, 0x44, 0x8c, 0x7e, 0x04, 0x2b, 0xe6, 0x23, 0x7d,
-	0xb6, 0x98, 0x61, 0x9c, 0xda, 0xe5, 0x55, 0xee, 0xbf, 0x6a, 0xb0, 0x55, 0x9d, 0x70, 0x5b, 0x30,
-	0xef, 0x38, 0xe3, 0xa7, 0xb0, 0x9c, 0xeb, 0xd7, 0xfb, 0xe4, 0x3d, 0xd7, 0xb4, 0x39, 0x78, 0xfb,
-	0xdf, 0x33, 0xf0, 0x7e, 0x49, 0x9b, 0xa3, 0x1e, 0xcc, 0x9f, 0x84, 0xb7, 0xfc, 0x86, 0xa2, 0x67,
-	0xf7, 0xe3, 0x99, 0xd4, 0x57, 0xf3, 0xc7, 0xf7, 0xb4, 0x4e, 0x92, 0xe3, 0x7e, 0x0f, 0xbd, 0x86,
-	0x85, 0xa4, 0xc9, 0x28, 0xfa, 0xe2, 0x5e, 0xd8, 0xe2, 0x28, 0x69, 0x7e, 0xf9, 0xb6, 0xb0, 0xf1,
-	0xda, 0xc7, 0xb0, 0xb6, 0xdf, 0xa7, 0xde, 0xcd, 0x59, 0xe6, 0x9f, 0x75, 0x68, 0x73, 0xaa, 0xff,
-	0x0e, 0x83, 0x48, 0x8d, 0x9a, 0x1f, 0x98, 0x65, 0xb2, 0xa6, 0x19, 0x4f, 0x7f, 0x04, 0x98, 0xa4,
-	0x1b, 0xdd, 0x8f, 0xd1, 0x54, 0x73, 0x36, 0x7f, 0xf2, 0xd6, 0xb8, 0x94, 0xc0, 0xde, 0x67, 0xdf,
-	0xb4, 0x7a, 0x4c, 0xf5, 0xe3, 0x2b, 0x0d, 0xda, 0x09, 0x98, 0x27, 0xb8, 0xe4, 0xd7, 0x6a, 0x27,
-	0xe0, 0xde, 0x8e, 0x88, 0xbc, 0x9d, 0xb1, 0xd3, 0x1d, 0xeb, 0xf4, 0x6a, 0xde, 0xc4, 0xf7, 0xfc,
-	0x7f, 0x01, 0x00, 0x00, 0xff, 0xff, 0x22, 0xb5, 0xde, 0x7e, 0x67, 0x15, 0x00, 0x00,
+	// 1752 bytes of a gzipped FileDescriptorProto
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0xb4, 0x58, 0xdd, 0x6e, 0x23, 0xb7,
```

Comment:
>x00, 0x00, 0x00, 0x00, 0x0 [](http://example.com/codeflow?start=20&length=26)

not sure what this is?

Reply:
I think this is the binary compressed protobuf. It gets autogenerated each time I make, pretty sure we need to check this in